### PR TITLE
Update CHANGELOG.md before release

### PR DESCRIPTION
**Pull request by dataclouder**
Update CHANGELOG.md from `.changes`

Note that the changes for current (and previous) release don't need to be removed.

### CODE REVIEW

**Review by vbauzysvmware — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,54 @@
...
+
+* `resource/vcd_vapp_vm` removed default value for `cpus` and `cpu_cores` [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* `resource/vcd_vapp_vm` fix ignoring `is_primary=false` [#556](https://github.com/vmware/terraform-provider-vcd/pull/556)
+* Fixed `vcd_independent_disk.size` issue, new field `size_in_mb` replaces the `size` [#588](https://github.com/vmware/terraform-provider-vcd/pull/588)
+
+NOTES
+
+* Added conditional skips for some checks in test `TestAccVcdVAppVmDhcpWait`
+* Dropped support for VCD 9.5 [#544](https://github.com/vmware/terraform-provider-vcd/pull/544)
+* Added support for VCD 10.2 [#544](https://github.com/vmware/terraform-provider-vcd/pull/544)
```
Comment (lvirbalas) [inline CHANGELOG.md:39]: Let's move these two a line up, so they appear first in the NOTES section.

**Review by Didainius — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,54 @@
...
-Changes in progress for v3.0.0 are available at [.changes/v3.0.0](https://github.com/vmware/terraform-provider-vcd/tree/master/.changes/v3.0.0) until the release.
+BUG FIXES
+
+* `resource/vcd_vapp_vm` removed default value for `cpus` and `cpu_cores` [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* `resource/vcd_vapp_vm` fix ignoring `is_primary=false` [#556](https://github.com/vmware/terraform-provider-vcd/pull/556)
+* Fixed `vcd_independent_disk.size` issue, new field `size_in_mb` replaces the `size` [#588](https://github.com/vmware/terraform-provider-vcd/pull/588)
+
+NOTES
+
+* Added conditional skips for some checks in test `TestAccVcdVAppVmDhcpWait`
```
Comment (lvirbalas) [inline CHANGELOG.md:37]: Let's move this to the bottom of this list, as tests are not very actual for users.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,54 @@
...
+* Added support for NSX-T Org VDC [#550](https://github.com/vmware/terraform-provider-vcd/pull/550)
+* `resource/vcd_org_vdc` new fields for assigning VM sizing policies `vm_sizing_policy_ids` and `default_vm_sizing_policy_id` [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* `resource/vcd_vapp_vm` new field `sizing_policy_id` uses VM sizing policy [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
 
-Changes in progress for v3.0.0 are available at [.changes/v3.0.0](https://github.com/vmware/terraform-provider-vcd/tree/master/.changes/v3.0.0) until the release.
+BUG FIXES
+
+* `resource/vcd_vapp_vm` removed default value for `cpus` and `cpu_cores` [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* `resource/vcd_vapp_vm` fix ignoring `is_primary=false` [#556](https://github.com/vmware/terraform-provider-vcd/pull/556)
+* Fixed `vcd_independent_disk.size` issue, new field `size_in_mb` replaces the `size` [#588](https://github.com/vmware/terraform-provider-vcd/pull/588)
```
Comment (lvirbalas) [inline CHANGELOG.md:33]: This is is effectively removal, so we should have a note in the REMOVALS section as well. What we want is that user opens up changelog and finds all breaking-changes to adapt to in a single section.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,54 @@
...
+* **New Resource**: `vcd_vm_sizing_policy` VDC VM sizing policy [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* **New Data Source**: `vcd_vm_sizing_policy` VDC VM sizing policy [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* **New Resource**: `vcd_edgegateway_settings` Changes LB and FW global settings for Edge Gateway [#557](https://github.com/vmware/terraform-provider-vcd/pull/557)
+* **New Resource**: `vcd_external_network_v2` with NSX-T support [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_external_network_v2` with NSX-T support [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_vcenter` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_portgroup` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_nsxt_manager` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_nsxt_tier0_router` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```
Comment (lvirbalas) [inline CHANGELOG.md:15]: Would be good to have a few words where they are used here in the changelog.
CC: @Didainius

**Review by lvirbalas — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (Didainius) [reply CHANGELOG.md:15]: All of these datasources are useful for `vcd_external_network_v2` to specify 

`vcd_vcenter` and `vcd_portgroup` are used to create NSX-V backed `vcd_external_network_v2`
`vcd_nsxt_manager` and `vcd_nsxt_tier0_router`  are used to create NSX-T backed `vcd_external_network_v2`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,54 @@
...
+FEATURES
+
+* **New Resource**: `vcd_vapp_access_control` Access control for vApps [#543](https://github.com/vmware/terraform-provider-vcd/pull/543)
+* **New Data Source**: `vcd_org_user` Org User [#543](https://github.com/vmware/terraform-provider-vcd/pull/543)
+* **New Resource**: `vcd_vm_sizing_policy` VDC VM sizing policy [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* **New Data Source**: `vcd_vm_sizing_policy` VDC VM sizing policy [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* **New Resource**: `vcd_edgegateway_settings` Changes LB and FW global settings for Edge Gateway [#557](https://github.com/vmware/terraform-provider-vcd/pull/557)
+* **New Resource**: `vcd_external_network_v2` with NSX-T support [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_external_network_v2` with NSX-T support [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_vcenter` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```
Comment (Didainius) [inline CHANGELOG.md:12]:
```suggestion
* **New Data Source**: `vcd_vcenter` useful for NSX-V backed `vcd_external_network_v2` resource [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,54 @@
...
+* **New Data Source**: `vcd_portgroup` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```
Comment (Didainius) [inline CHANGELOG.md:13]:
```suggestion
* **New Data Source**: `vcd_portgroup` useful for NSX-V backed `vcd_external_network_v2` resource [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,54 @@
...
+* **New Data Source**: `vcd_nsxt_manager` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```
Comment (Didainius) [inline CHANGELOG.md:14]:
```suggestion
* **New Data Source**: `vcd_nsxt_manager` useful for NSX-T backed `vcd_external_network_v2` resource [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```

File: CHANGELOG.md (hunk 4 above)
Comment (Didainius) [inline CHANGELOG.md:15]:
```suggestion
* **New Data Source**: `vcd_nsxt_tier0_router` useful for NSX-T backed `vcd_external_network_v2` resource [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,6 +1,54 @@
...
+
+REMOVALS
+
+* Removed deprecated resource `vcd_network` [#543](https://github.com/vmware/terraform-provider-vcd/pull/543)
+* Removed deprecated resources `vcd_dnat`, `vcd_snat`, and `vcd_firewall_rules` [#557](https://github.com/vmware/terraform-provider-vcd/pull/557)
+* Removed deprecated attributes `ip, network_name, vapp_network_name, network_href, mac, initscript` from `vcd_vapp_vm` [#563](https://github.com/vmware/terraform-provider-vcd/pull/563)
+* Removed deprecated attributes `external_networks, default_gateway_network, advaced` from `vcd_edgegateway` [#588](https://github.com/vmware/terraform-provider-vcd/pull/588)
+* Removed `vcd_independent_disk.size` in favor of `vcd_independent_disk.size_in_mb` [#588](https://github.com/vmware/terraform-provider-vcd/pull/588)
+* Removed deprecated attributes `template_name, catalog_name, network_name, memory, cpus, ip, storage_profile, initscript, ovf, accept_all_eulas` from `vcd_vapp` [#588](https://github.com/vmware/terraform-provider-vcd/pull/588)
+* Fixed `vcd_independent_disk.size` issue, new field `size_in_mb` replaces the `size` [#588](https://github.com/vmware/terraform-provider-vcd/pull/588)
```
Comment (lvirbalas) [inline CHANGELOG.md:51]: Let's also move this one to the top of the list, so "Fixed" comes first and then all the "Removed" lines follow.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,6 +1,54 @@
...
+* **New Data Source**: `vcd_vm_sizing_policy` VDC VM sizing policy [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* **New Resource**: `vcd_edgegateway_settings` Changes LB and FW global settings for Edge Gateway [#557](https://github.com/vmware/terraform-provider-vcd/pull/557)
+* **New Resource**: `vcd_external_network_v2` with NSX-T support [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_external_network_v2` with NSX-T support [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_vcenter` useful for `vcd_external_network_v2` resource when used with NSX-V [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_portgroup` useful for `vcd_external_network_v2` resource when used with NSX-V [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_nsxt_manager` useful for `vcd_external_network_v2` resource when used with NSX-V [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```
Comment (Didainius) [inline CHANGELOG.md:14]:
```suggestion
* **New Data Source**: `vcd_nsxt_manager` useful for `vcd_external_network_v2` resource when used with NSX-T [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,6 +1,54 @@
...
+* **New Data Source**: `vcd_nsxt_tier0_router` useful for `vcd_external_network_v2` resource when used with NSX-V [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```
Comment (Didainius) [inline CHANGELOG.md:15]:
```suggestion
* **New Data Source**: `vcd_nsxt_tier0_router` useful for `vcd_external_network_v2` resource when used with NSX-T [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```

**Review by lvirbalas — APPROVED** [review]

**Pull request merged**